examples/simple_conduction/with_input_file.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/**
+ * @file with_input_file.cpp
+ *
+ * @brief A simple example of steady-state thermal conduction that uses
+ * a Lua input file to configure the simulation
+ */
+
+#include "serac/physics/thermal_conduction.hpp"
+#include "serac/infrastructure/initialize.hpp"
+#include "serac/infrastructure/terminator.hpp"
+#include "serac/numerics/mesh_utils.hpp"
+#include "serac/serac_config.hpp" // for SERAC_REPO_DIR
+
+const auto input_file = SERAC_REPO_DIR "/examples/simple_conduction/conduction.lua";
+
+int main(int argc, char* argv[])
+{
+  /*auto [num_procs, rank] = */serac::initialize(argc, argv);
```

Comment:
Did you leave commented out code here for a reason?

Reply:
I'm torn between my hatred of commented code vs useful information in the tutorial. @jamiebramwell ?

Reply:
I think it's worth keeping as someone may want this information in their code.